File: datalib/src/main/java/nl/acidcats/tumblrlikes/core/usecases/lifecycle/AppLifecycleUseCase.java
package nl.acidcats.tumblrlikes.core.usecases.lifecycle;

import rx.Observable;

/**
 * Created on 17/10/2018.
 */
public interface AppLifecycleUseCase {

    Observable<Boolean> setAppStopped(long currentTimeMs);

    Observable<Boolean> isAppStoppedTooLong(long currentTimeMs);
}

File: datalib/src/main/java/nl/acidcats/tumblrlikes/core/usecases/lifecycle/AppLifecycleUseCase.kt
package nl.acidcats.tumblrlikes.core.usecases.lifecycle

import rx.Observable

/**
 * Created on 24/10/2018.
 */
interface AppLifecycleUseCase {
    fun setAppStopped(currentTimeMs: Long): Observable<Boolean>

    fun isAppStoppedTooLong(currentTimeMs: Long): Observable<Boolean>
}